projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -342,18 +342,10 @@ static void validate(Map<String, String> parameters,
                try {
                   validateType(value, variable);
                } catch (BatfishException e2) {
-                  String messageHead
-                        = String.format("Error when validating parameter: %s: ",
-                        parameterName);
-                  String messageBody
-                        = String.format("The value: %s, is a type " +
-                        "of %s, doesn't match the type expected for %s: %s",
-                        parameterValue, value.getNodeType(), parameterName,
-                        variable.getType());
                   String errorMessage
-                        = messageHead.concat(e2.getMessage().isEmpty() ?
-                                             messageBody : e2.getMessage());
-                  throw new BatfishException(errorMessage);
+                        = String.format("Invalid value: %s for parameter: %s: ",
```

Comment:
Please drop all the colons; extra punctuation just breaks up the flow of the text, makes it harder to read and less grammatically correct.

Instead, if you're concerned about it being hard to tell where the value begins and ends:

```java
Invalid value for parameter %s: [%s].   // parameter name then value, since we know the name is small doesn't have any special characters.
```

note that you don't need the final colon on the string either, it shows up naturally in the way that Java stack traces are rendered.